lib/matplotlib/ticker.py

```diff
@@ -1872,7 +1914,13 @@ def tick_values(self, vmin, vmax):
         return self.raise_if_exceeds(np.array(ticklocs))
 
 
-class AutoLocator(MaxNLocator):
```

Comment:
I am 50/50 on doing the renaming like this.  If we black list setting `nbins` in `set_params` this will break user code (like an SO question I wrote this morning).